### ISSUE TITLE
DeepMedic

### ISSUE THREAD

**Issue by alonshmilo** [alonshmilo/MedicalData_jce#27]
# Issue: DeepMedic

### Explanation:
Efficient Multi-Scale 3D Convolutional Neural Network for Brain Lesion Segmentation. Understanding the network, making it work on our macs, and start to think about the rib cage bone orientation.

### Checklist:
- [x] Code
- [x] Reading
- [x] Documenting reading
- [x] Project book
- [x] Advisor consult

**Comment by alonshmilo** [comment]
DeepMedic installed on mac

**Comment by alonshmilo** [comment]
Writing and explaining - myself
Coding (using our data) and configurations - myself & @stavdv

**Comment by alonshmilo** [comment]
Working on configuration files and training sets

**Comment by stavdv** [comment]
There are three kind of configuration file to describe our data:
1. label - to mark only the bones.
2. mask - 1: if we have bone in that place, and 0 otherwise.
3. channel- CT scan.

**Comment by stavdv** [comment]
Folder for deep medic
@alonshmilo make calculations for configuration file

**Comment by alonshmilo** [comment]
@stavdv Should be same if [25,25,25) is the vector for input.

**Comment by stavdv** [comment]
Ok, configuration is done

**Comment by stavdv** [comment]
done 👍
